spellmonitor.lic

```diff
@@ -110,8 +110,10 @@ spell_action = proc do |server_string|
   end
 
   # The start of one or more active spells!
-  if data =~ %r{^<pushStream id="percWindow"/>}
+  if data =~ %r{<pushStream id="percWindow"/>}
     DRSpells.grabbing_spell_data = true
+    index = data.index("<pushStream id=\"percWindow\"/>")
+    data.slice!(index, index+29).strip!
```

Comment:
The start of the slice still needs to be `0` to take care of getting past any preceeding text before the `<pushStream id="percWindow"/>`
```suggestion
    data.slice!(0, index + 29).strip!
```